src/ceph-disk/ceph_disk/main.py

```diff
@@ -74,6 +84,14 @@
             'ready': 'cafecafe-9b03-4f30-b4c6-35865ceff106',
             'tobe': '89c57f98-2fe5-4dc0-89c1-35865ceff2be',
         },
+        'block.db': {
+            'ready': '166418da-c469-4022-adf4-b30afd37f176',
+            'tobe': '7521c784-4626-4260-bc8d-ba77a0f5ff97',
+        },
+        'block.wal': {
+            'ready': '86a32090-3647-40b9-bbbd-38d8c573aa86',
+            'tobe': '92dad30f-175b-4d40-a5b0-5c0a258b4d28',
```

Comment:
Unfortunately I noticed the cute **2be** at the end after adding the [mpath]*[tobe] ones. And now they can't be changed otherwise that may impact partitions with an existing tobe UUID.